README.md

```diff
@@ -1,86 +1,97 @@
+and [Tensorflow Lite](https://www.tensorflow.org/lite)
+custom Ops for specific instruction sets, developed in inline assembly or in C++ 
+using compiler intrinsics. LCE leverages optimization techniques
+such as **tiling** to maximize the number of cache hits, **vectorization** to maximize 
+the computational throughput, and **multi-threading parallelization** to take
+advantage of multi-core modern desktop and mobile CPUs.
+
+## Key Features
+- Tight integration of LCE with [Larq](https://larq.dev) and
+  TensorFlow provides a smooth end-to-end training and deployment experience.
+- LCE enables high performance, on-device machine learning inference by
+  providing hand-optimized kernels and network level optimizations for BNN models.
+- LCE currently supports ARM64-based mobile platforms such Android phones
+  and the Raspberry Pi.
+- Thread parallelism support in LCE is essential for modern mobile devices with
+  multi-core CPUs.
+- A collection of Larq pre-trained BNN models for common machine learning tasks
+  is available in [Larq Zoo](https://github.com/larq/zoo)
+  and can be used seamlessly with LCE.
+- LCE provides a custom [MLIR-based model converter](./lce_converter.md) which
+  is fully compatible with Tensorflow Lite and performs additional
+  network level optimizations for Larq models.
+
+## Performance
+The table below presents **single-threaded** performance of Larq Compute Engine on multiple
+generations of Larq BNN models one the [Pixel phone (2016)](https://support.google.com/pixelphone/answer/7158570?hl=en-GB)
+and (Raspberry Pi 4 [BCM2711](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711/README.md)) board:
+
+| Model         | Accuracy  | Pixel, ms   | RPi 4 (BCM2711), ms |
+| ------------- | :-------: | :---------: | :----------:        |
+| TODO          | TODO      | TODO        | TODO                |
+| TODO          | TODO      | TODO        | TODO                |
+| TODO          | TODO      | TODO        | TODO                |
+| TODO          | TODO      | TODO        | TODO                |
+
+The following table presents **multi-threaded** performance of Larq Compute Engine on
+a Pixel 1 phone and a Raspberry Pi 4 board:
+
+| Model              | Accuracy  | Pixel, ms   | RPi 4 (BCM2711), ms |
+| ------------------ | :-------: | :---------: | :----------:        |
+| TODO               | TODO      | TODO        | TODO                |
+| TODO               | TODO      | TODO        | TODO                |
+| TODO               | TODO      | TODO        | TODO                |
+| TODO               | TODO      | TODO        | TODO                |
+
+Benchmarked on February, TODO with LCE custom
+[TFLite Model Benchmark Tool](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/tools/benchmark)
+(see [here](../larq_compute_engine/tflite/benchmark))
+and BNN models with randomized weights and inputs.
+
+## Getting started
```

Comment:
This section now overlaps a lot with the `get_started.md` document. So I think we should merge this info into the other document and just give the link to that doc here.